src/components/Channel/hooks/useMentionsHandlers.ts

```diff
@@ -0,0 +1,52 @@
+import { MouseEvent, useCallback } from 'react';
+import type { UserResponse } from 'stream-chat';
```

Comment:
also does `MouseEvent` need to be imported here individually because `React` default isn't being pulled off of this import?
```suggestion
import { MouseEvent, useCallback } from 'react';

import type { UserResponse } from 'stream-chat';
```

Reply:
yep no jsx so no need for React

Reply:
yeah I got the lack of need for React but I meant is importing MouseEvent necessary because the default isn't being imported because otherwise it's a natural type that is present